### PR TITLE
feat: Portuguese

### PR DESCRIPTION
**Pull request by HavardNot**
adds portuguese language support for mad-expo-core

### CODE REVIEW

**Review by dansvend — COMMENTED** [review]
Nice with a cleanup of unused translations and the addition of Portuguese 👌 I left a couple of comments you might want to look into 🙌

File: src/resources/language/dictionary.ts [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
 import en from "./en.json";
 import no from "./no.json";
+import pt from "./pt.json";
 
 export type DictionaryObject = typeof en;
 
 const dictionaries: Record<string, DictionaryObject> = {
   en,
   no,
+  pt
```
[the remark this answers is not in the document]
Comment (HavardNot) [reply src/resources/language/dictionary.ts:11]: added pt to DoneButtonInputAccessoryView and ServiceMessage

File: src/resources/language/pt.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  "settings.loggedInAs": "Conectado com",
+  "settings.language": "Idioma",
+  "settings.logOut": "Sair",
+  "settings.back": "Voltar",
+  "Language": "Idioma",
+  "English": "Inglês",
+  "Norwegian": "Norueguês",
+  "Submit": "Enviar",
+  "keyboard.done": "Feito",
+  "serviceMessage.couldNotRetrieve": "Não foi possível recuperar a mensagem de serviço"
```
[the remark this answers is not in the document]
Comment (dansvend) [reply src/resources/language/pt.json]: English and Norwegian have the properties in the same order, but Portguese still have a couple of properties in the wrong order.

**Review by dansvend — COMMENTED** [review]
Really close 💪 Only a couple of properties in the wrong order in the Portuguese dictionary left 🚀

**Review by dansvend — APPROVED** [review]
Looks good to me - nice job 💪 Let's get this shipped 🚀

**Pull request merged**